testeng/jobs/scanPluginUpdatesJob.groovy

```diff
@@ -0,0 +1,52 @@
+package testeng
+
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_LOG_ROTATOR
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_HIPCHAT
+
+job('scan-plugin-updates') {
+
+    description(
+        'Scan the plugins installed on this Jenkins instance for security related updates'
+    )
+    authorization {
+        blocksInheritance(true)
+        permissionAll('edx*testeng')
+    }
+
+    logRotator JENKINS_PUBLIC_LOG_ROTATOR()
+    label('jenkins-worker')
+    concurrentBuild(false)
+
+    scm {
+        git {
+            remote {
+                url('https://github.com/edx/jenkins-configuration.git')
+            }
+            branch('*/master')
+            browser()
```

Comment:
What does the repository browser gain us in this case?

Reply:
This is a carryover from before we upgraded to Jenkins 2.x and all of the newer plugin versions. In the past, there was a bug that would cause an exception on jobs using git SCM w/o this in the DSL. I just checked, and it is no longer the case, so we can take it out